alacritty_terminal/src/term/mod.rs

```diff
@@ -1154,6 +1362,8 @@ impl<T> Term<T> {
 
         // Recreate tabs list
         self.tabs = TabStops::new(self.grid.num_cols(), self.tabspaces);
+        self.damage.resize_damage(self.grid.num_lines(), self.grid.num_cols());
+        self.damage.damage_everything();
```

Comment:
Just set `full_damage` directly, since you're already doing it. Also `damage_all` should be a better name. What do you think?